scripts/create-sentry-release

```diff
@@ -31,10 +31,15 @@ docker pull "${DOCKER_IMAGE}"
 
 docker run --rm --entrypoint cat "${DOCKER_IMAGE}" /opt/relay-debug.zip > relay-debug.zip
 docker run --rm --entrypoint cat "${DOCKER_IMAGE}" /opt/relay.src.zip > relay.src.zip
+docker run --rm --entrypoint tar "${DOCKER_IMAGE}" -cf - /lib/x86_64-linux-gnu > libs.tar
+tar xf libs.tar
 
 echo 'Uploading debug information and source bundle...'
 sentry-cli upload-dif ./relay-debug.zip ./relay.src.zip
 
+echo 'Uploading system symbols...'
+sentry-cli upload-dif lib/x86_64-linux-gnu
```

Comment:
can't sentry-cli walk tarballs already?

Reply:
Apparently no:
```
$ sentry-cli upload-dif libs.tar
> Found 0 debug information files
> No debug information files found
```
Also, there is no `zip` command in our docker image.